cmd/juju/commands/main_test.go

```diff
@@ -574,6 +574,7 @@ var commandNames = []string{
 	"set-plan",
 	"set-series",
 	"set-wallet",
+	"show-action",
```

Comment:
Wouldn't this be conditional based on whether a feature flag is on or not?

Reply:
No, this command is not behind a feature flag, it's not reliant on juju 3